graphics/include/ignition/common/Material.hh

```diff
@@ -177,6 +177,14 @@ namespace ignition
       /// \return The enable two sided rendering value
       public: bool TwoSidedEnabled() const;
 
+      /// \brief Set the render order
+      /// \param[in] _t The render order value
+      public: void SetRenderOrder(int _renderOrder);
+
+      /// \brief Get the render order
```

Comment:
Could you document the default value? Also, I'm wondering what a negative value means, is that allowed?